src/main/java/billgates/database/MySQLDatabaseGateway.java

```diff
@@ -366,6 +487,34 @@ currency         CHAR(3)         NOT NULL,
         }
     }
 
+    @Override
+    public void createSplitBillTable(int billId) {
+        try {
+            Statement statement = connection.createStatement();
+
+            String query = String.format("""
```

Comment:
The create table statement for creating a split bill table should not the same as creating a usual one.

Refers to this issue. #11